README.md

```diff
@@ -21,11 +21,11 @@ More generally, this project might be for you if you believe...
 - Epics (backed by redux-observable) as mechanism for side effects/middleware (vs. thunks or sagas)
 - Observables are pretty dope
 
-## **Philosophy**
+## Philosophy
 
-Redux is an event dispatching system that operates on a single global state atom. It makes little sense to begin building your application before defining two interfaces: the shape of your state and the shape of all actions that operate on that state. From there we can leverage the strict unidirectional flow of Redux in our type system.
+Redux is an event dispatching system that operates on a single global state atom that is only modifiable via actions. It makes little sense to begin building your application before defining two interfaces: the shape of your state and the shape of all actions that operate on that state. From there we can leverage the strict unidirectional flow of Redux in our type system.
 
-You should be familiar with the basic terminology of Redux (Actions, Reducers, Action Creators, Stores, Middleware) before reading further. [Their documentation](https://redux.js.org/) is excellent, so we'd strongly encourage reading it.
+You should be familiar with the basic terminology of Redux (Actions, Reducers, Action Creators, Stores, Middleware) and RxJS (Observables, Epics) before reading further. The documentation for [Redux](https://redux.js.org/) and [RxJS](https://rxjs-dev.firebaseapp.com/guide/overview) is excellent, so we'd strongly encourage reading it.
```

Comment:
Probably makes more sense to reference redux-observable docs - https://redux-observable.js.org/docs/basics/Epics.html.

This is where Epics come from